Source/Runes.swift

```diff
@@ -50,3 +50,27 @@ infix operator -<< {
     // operator (`=`)
     precedence 100
 }
+
+/**
+compose two functions that produce results in a context, from left to right, returning a result in that context
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 111 characters (line_length)